gitinstall-hp.md

```diff
@@ -0,0 +1,62 @@
+# Installing git on HP-UX
+
+## Using Depot packages
+
+* Create a temporary folder to hold the depot files : `mkdir -p /root/GIT`
+* Copy over the depot packages. [HPv3 ia64](\\redmond\wsscfs\OSTCData\NixArchive\SourceCodeManagement\git\HP-UX\v3ia64), [HPv3 parisc](\\redmond\wsscfs\OSTCData\NixArchive\SourceCodeManagement\git\HP-UX\v3risc)
```

Comment:
This link doesn't appear to work. As I recall, I had problems with this too. If you can't get it to work, then put it in the document with a `\path\like\this`.